example/web-access-control/protected-operation/acl-propagation.feature

```diff
@@ -40,20 +39,16 @@ Feature: Inheritable ACL controls child resources
     And headers clients.bob.getAuthHeaders('PUT', resource2.getUrl())
     And header Content-Type = 'text/plain'
     When method PUT
-    Then status 201
+    Then match [200, 201, 204, 205] contains responseStatus
```

Comment:
Actually, that test is correct, only `201` is allowed by HTTP when a resource is created.

Reply:
The same change was incorrectly made to the specification-tests version recently so I will revert that too and restore the comment that explained why 201 was mandated for a created resource